data/json/monsters/zed_worm.json

```diff
@@ -0,0 +1,243 @@
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "GRABS",
+      "NO_BREATHE",
+      "PUSH_MON",
+      "FILTHY",
+      "RANGED_ATTACKER"
+    ]
+  },
+  {
+    "id": "mon_zombie_worm_brute",
+    "type": "MONSTER",
+    "name": { "str": "spindly carrier" },
+    "description": "A zombie with a hunched, ape-like posture supported by surprisingly thin arms.  On the back itself resides an unnaturally large tapeworm, snaking around and twisting its hooked head.  Several smaller worms can be seen throughout the wounds on the body, the most developed ones wrapping themselves around the arms.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 200,
+    "speed": 80,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "red",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 4,
+    "melee_dice_sides": 5,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "dodge": 2,
+    "armor_bash": 3,
+    "armor_cut": 3,
+    "armor_elec": 1,
+    "bleed_rate": 0,
+    "vision_night": 5,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_wp_zombie" ],
+    "harvest": "zombie",
+    "special_attacks": [
+      [ "RANGED_PULL", 75 ],
+      { "type": "bite", "cooldown": 5 },
+      [ "scratch", 20 ],
+      {
+        "id": "tapeworm",
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 14, "armor_penetration": 4 } ],
+        "range": 7
+      }
+    ],
+    "death_function": { "effect": { "id": "death_tapeworm", "hit_self": true } },
+    "death_drops": "mon_zombie_hulk_death_drops",
+    "zombify_into": "mon_zombie_grappler",
+    "burn_into": "mon_zombie_scorched",
+    "fungalize_into": "mon_zombie_fungus",
+    "flags": [
+      "SEES",
+      "HEARS",
+      "SMELLS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "NO_BREATHE",
+      "PUSH_MON",
+      "FILTHY",
+      "RANGED_ATTACKER"
+    ]
+  },
+  {
+    "id": "mon_zombie_worm_hulk",
+    "type": "MONSTER",
+    "name": { "str": "twisted hive" },
+    "description": "An unnaturally large emaciated quadrupedal creature that you can only assume has once been human.  The actual host body is almost skeletal, visibly starved of any excess biomass to feed the evergrowing mass or mutant worms inside.  The forelimbs are visibly elongated and end in long bony claws, while the hindlimbs have developed a digitigrade foot structure.  On its torso, both from the top and from the bottom, there are snaking dozens if not hundreds of worms of various sizes and on the central part of the back resides the largest one, coiled several times around the creature.",
+    "default_faction": "zombie",
```

Comment:
Why would a "zombie" that's mostly the mutant parasites(which should be hostile to zombies) still be the same faction 🤔

>Since zombos now have a reason to hunt bugs dissolve the zombie / creepy crawly truce and watch swamps neighboring towns erupt into a multigenerational free-for-all. That should be fun. ~ #60605

Reply:
The idea here was something akin to a Plague Vector, which is somewhere in the middle between a zombie and a mutant due to the fact it fed on enough 'tainted' flesh. I'm not actually sure where Plague Vectors fall on the faction chart now after #60605, though, and I have to check that.